Dockerfile

```diff
@@ -1,13 +1,20 @@
 # Build on top of a Node.js + npm image.
-FROM node:latest
+FROM node:10
 
 # Make a codebase directory and work in it
 RUN mkdir codebase
 WORKDIR /codebase
 
 # Load the package.json and package-lock.json files
-COPY package.json .
-COPY package-lock.json .
+COPY package*.json ./
```

Comment:
[Explicit is better than implicit. Simple is better than complex.](https://www.python.org/dev/peps/pep-0020/#the-zen-of-python)

Reply:
> [Explicit is better than implicit. Simple is better than complex.](https://www.python.org/dev/peps/pep-0020/#the-zen-of-python)

I'll definitely remember this :smile: